frontend/src/components/TemplatesIndex/TemplatesIndex.tsx

```diff
@@ -51,84 +59,11 @@ export const TemplatesIndex: FC<TemplateIndexProps> = () => {
+        'name', 'updatedAt', 'createdBy', 'labName', 'status'
     ]
-    const templates2: Template[] = [
 
-        {
-            name: "MumpsQuestV1",
-            lab: "Quest",
-            createdBy: "J.Smith",
-            status: "Completed",
-            lastUpdated: new Date(Date.parse("2025-03-24T12:00:00.000-05:00"))
-        },
-        {
-            name: "LBTIRadar",
-            lab: "Radar",
-            createdBy: "C.Alex",
-            status: "Completed",
-            lastUpdated: new Date(Date.parse("2025-05-30T12:00:00.000-05:00"))
-        },
-        {
-            name: "COVIDBaylor1",
-            lab: "Emory",
-            createdBy: "A.Bryant",
-            status: "Completed",
-            lastUpdated: new Date(Date.parse("2025-06-21T12:00:00.000-05:00"))
-        },
-        {
-            name: "COVIDEMory",
-            lab: "Baylor",
-            createdBy: "D.Smith",
-            status: "Completed",
-            lastUpdated: new Date(Date.parse("2024-06-21T12:00:00.000-05:00"))
-        },
-    ];
-
-    // TODO: These are just for demo purposes and allow easily manipulating the template store
-    useEffect(() => {
-        console.debug(`
-        The following methods have been added to the window:
-        
-        LoadNiceTemplates - this will load some pre-formatted templates that display nicely
-        SaveTemplates - this will save the current templates to 'oldTemplates'
-        LoadSavedTemplates - this will load the templates saved in 'oldTemplates'
-        ClearTemplates - this will delete the current templates 
-        
-        `)
-        window.LoadNiceTemplates = () => {
-            const oldTemplates = localStorage.getItem('templates')
-            localStorage.setItem('oldTemplates', oldTemplates)
-            localStorage.setItem('templates', JSON.stringify(templates2))
-            setTemplates(templates2)
-        }
-        window.SaveTemplates = () => {
-            const oldTemplates = localStorage.getItem('templates')
-            if (!oldTemplates) {
-                return
-            }
-            localStorage.setItem('oldTemplates', oldTemplates)
-        }
-        window.ClearTemplates = () => {
-            localStorage.removeItem('templates')
-            setTemplates([])
-        }
-        window.LoadSavedTemplates = () => {
-            const oldTemplates = localStorage.getItem('oldTemplates')
-            if (oldTemplates) {
-                localStorage.setItem('templates', oldTemplates)
-                setTemplates(JSON.parse(oldTemplates))
-            }
-        }
-        return () => {
-            delete window.LoadNiceTemplates
-            delete window.SaveTemplates
-            delete window.ClearTemplates
-            delete window.LoadSavedTemplates
-        }
-    });
 
-    if (!templates || templates.length === 0) {
+    if (!templateQuery.data  || templateQuery.data.length === 0) {
         return (
             <><img
```

Comment:
`<p>No templates available.</p>`

Reply:
I was trying to avoid making UI changes and instead of just change the underlying data.